internal/time/time.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package time
+
+import "time"
+
+func ParseTimestamp(timestamp string) (time.Time, error) {
+	layouts := []string{
```

Comment:
I think we look for 2 new formats where the timezone is hardcoded 
```
YYYY-MM-DDTHH:MM:SS-0700
YYYY-MM-DDTHH:MM:SSZ07:00
```

In this format timezone is hardcoded. Meaning:
 "0700" indicating the time's offset from UTC in hours and minutes.

We need to generalizethat.. but that leads us to the standard time format (with and without zulu)
![image](https://github.com/mongodb/mongodb-atlas-cli/assets/981838/2efbab70-e1ea-41bb-b084-b4c971d86eb7)